src/backend/api/library.ts

```diff
@@ -113,6 +117,11 @@ export const onLibraryChange = async (
   }
 }
 
+export const getGameOverride = async () => ipcRenderer.invoke('getGameOverride')
+
+export const getGameSdl = async (appName: string) =>
```

Comment:
maybe I am a n00b but it took me some searching to figure out this was selective downloads. maybe we could write that out for this variable name to make it more clear

Reply:
Yeah, this is how Legendary call it so we keep it, but I can rename it then.

Reply:
Added some comments and JSDOC to make it more clear.